servicenow/check.py

```diff
@@ -0,0 +1,199 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+
+# 3rd party
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from requests.packages.urllib3 import disable_warnings
+
+# Disable https warnings
+disable_warnings(InsecureRequestWarning)
```

Comment:
What does this mean? Are we suppressing certificate checking? I don't think that is very secure

Reply:
true, we are suppressing the warning coming from unverified HTTPS request made.

Reply:
lets dicuss with @markbakker78 whether this is acceptable

Reply:
lets dicuss with @markbakker78 whether this is acceptable